### DISCUSSION TITLE
Error while adding netlify cms

### DISCUSSION THREAD

**Discussion by castasint** [gatsbyjs/gatsby#31441] [Help]
I am trying to add netlify cms plugin(using ```npm install netlify-cms-app gatsby-plugin-netlify-cms``` but I am getting the below error. I'm following this post [https://www.gatsbyjs.com/docs/how-to/sourcing-data/sourcing-from-netlify-cms/](https://www.gatsbyjs.com/docs/how-to/sourcing-data/sourcing-from-netlify-cms/).   Any advice on how it can be fixed?

![image](https://user-images.githubusercontent.com/16588875/118277228-320b2100-b4e6-11eb-8e22-efbca726860d.png)

Package.JSON: 

![image](https://user-images.githubusercontent.com/16588875/118277390-5ebf3880-b4e6-11eb-961d-04ba9949a5c2.png)

**Comment by LekoArts** [comment]
Try running the installation with the `--legacy-peer-deps` flag as suggested in the message
